setup.py

```diff
@@ -111,6 +111,7 @@ def get_version(version_file):
         "mongo-db-from-config@http://github.com/cisagov/mongo-db-from-config/tarball/develop",
         "openpyxl",
         "pandas == 1.1.5",
+        # "psycopg2 == 2.9.3",
```

Comment:
If this package is no longer needed, it should be deleted, rather than commented out.

Reply:
@dav3r , 

I have addressed this in commit [c444838](https://github.com/cisagov/pe-reports/pull/169/commits/c44483820f717e921da82ab223e252b6a04744d9)